relay_benchmark_test.go

```diff
@@ -223,3 +225,20 @@ func BenchmarkRelay2Servers5Clients4k(b *testing.B) {
 	p.servers = 2
 	benchmarkRelay(b, p)
 }
+
+func BenchmarkRelayWithAppends(b *testing.B) {
+	n, err := strconv.Atoi(os.Getenv("BENCH_RELAY_APPENDS_N"))
+	require.NoError(b, err)
+
+	p := defaultParams()
+	for i := 0; i < n; i++ {
+		p.appends = append(p.appends, relay.KeyVal{Key: []byte("foo"), Val: []byte("bar")})
+	}
+	b.ResetTimer()
+	benchmarkRelay(b, p)
+}
+
+func BenchmarkRelayWithoutAppends(b *testing.B) {
```

Comment:
I initially had to check why this was different than the 0 case in `BenchmarkRelayWithAppends`, then realized the 0 case was actually 1 append